From 534e50e45d9c7b55d58021322952f6e9cf7bb958 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 26 Jul 2013 10:09:58 +0800
Subject: [PATCH 220/602] ENGR00215174 add TARGET_CPU_VARIANT in imx6 and imx5
 platform.

add TARGET_CPU_VARIANT in imx6 and imx5 platform.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx5x/BoardConfigCommon.mk | 1 +
 imx6/BoardConfigCommon.mk  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index 5c4c230..50003c6 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -9,6 +9,7 @@ TARGET_CPU_ABI2 := armeabi
 TARGET_CPU_SMP := false
 TARGET_ARCH := arm
 TARGET_ARCH_VARIANT := armv7-a-neon
+TARGET_CPU_VARIANT := cortex-a8
 ARCH_ARM_HAVE_TLS_REGISTER := true
 ARCH_ARM_USE_NON_NEON_MEMCPY := false
 
diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 5125b0a..db45d85 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -7,6 +7,7 @@ TARGET_CPU_ABI := armeabi-v7a
 TARGET_CPU_ABI2 := armeabi
 TARGET_ARCH := arm
 TARGET_ARCH_VARIANT := armv7-a-neon
+TARGET_CPU_VARIANT := cortex-a9
 ARCH_ARM_HAVE_TLS_REGISTER := true
 
 TARGET_NO_BOOTLOADER := true
-- 
1.8.0

